daemon-scheduler/pkg/store/environment_test.go

```diff
@@ -192,3 +213,24 @@ func (suite *EnvironmentTestSuite) TestList() {
 		assert.Contains(suite.T(), envs, expectedEnv, "Expected %s to be returned by ListEnvironments", expectedEnv)
 	}
 }
+
+func (suite *EnvironmentTestSuite) TestDeleteEnvironmentNoName() {
+	suite.datastore.EXPECT().Delete(gomock.Any(), gomock.Any()).Times(0)
+
+	err := suite.environmentStore.DeleteEnvironment(suite.ctx, "")
+	assert.NotNil(suite.T(), err, "Expected an error when deleting an environment with no name")
```

Comment:
Sure, I think I've used NotNil() pretty much everywhere. I'll walk through the code and update.